src/app/Table.elm

```diff
@@ -1,29 +1,91 @@
-module Table exposing (view, init, Msg)
+module Table exposing (view, init, initWithPlayersAndCup)
+
+{-|
+
+   This module handles the table of players,
+   and everything relating to that.
+
+-}
 
 import Html exposing (..)
+import Html.Events exposing (..)
+import Types exposing (PlayerID)
+import TableMsg exposing (Msg(..))
+import Dice
 
 
-{-
+type alias PlayerName =
```

Comment:
Yeah, maybe. I didn't want to prescriptively change everything in `Types.elm`. I'm trying to make minimal conflicting changes to things other than the `Table` module. ;-) 👍 this is so we can all work together on it at the same time 👍